packages/core/src/transport/httpRequest.ts

```diff
@@ -72,16 +70,15 @@ function sendBeaconStrategy(endpointBuilder: EndpointBuilder, bytesLimit: number
     }
   }
 
-  const xhrUrl = endpointBuilder.build('xhr', payload)
-  sendXHR(xhrUrl, payload.data)
+  fetchStrategy(endpointBuilder, payload)
 }
 
 let hasReportedBeaconError = false
 
 function reportBeaconError(e: unknown) {
   if (!hasReportedBeaconError) {
     hasReportedBeaconError = true
-    addTelemetryError(e)
+    monitorError(e)
```

Comment:
`monitorError()` will send telemetry but will also log debug message in the console  (when `DD_RUM._setDebug(true)`)